drivers/hmis/app/graphql/types/forms/pick_list_option.rb

```diff
@@ -129,6 +129,14 @@ def self.static_options_for_type(pick_list_type, user:)
         client_audit_event_record_type_picklist
       when 'PROJECTS_RECEIVING_REFERRALS'
         projects_receiving_referrals
+      when 'FORM_TYPES'
```

Comment:
Thanks @gigxz you are right this needs to be a backend picklist. I'm not sure I have totally done this right; I wanted to reuse the logic about capitalizing 'CE' which seems to live in `use_enum_with_same_key`

Reply:
This seems right. Can you add a comment explaining why the extra permission is checked?